yarpctest/recorder/recorder_test.go

```diff
@@ -349,7 +349,7 @@ func TestReplaying(t *testing.T) {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 		defer cancel()
 
-		rbody, _, err := client.Call(ctx, yarpc.NewReqMeta().Procedure("hello"), []byte("Hello"))
+		rbody, err := client.Call(ctx, "hello", []byte("Hello"))
```

Comment:
I actually find this API much better than having the Procedure, a required param, ride the reqMeta. That always felt really weird.